setup.py

```diff
@@ -42,13 +42,14 @@ def get_version(rel_path):
         "License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)",
         "Operating System :: OS Independent",
     ],
-    python_requires=">=3.6",
+    python_requires=">=3.7",
     install_requires=[
         "scriptengine>=0.8.1",
         "pyYAML>=5.1",
-        "numpy>=1.16.1",
+        "matplotlib>=3.1",
+        "numpy>=1.18",
         "imageio>=2.0",
-        "scitools-iris>=3.1",
+        "scitools-iris>=3.1,!=3.2.0,!=3.2.0.post0",
```

Comment:
Why is that ending up here? Didn't we fix that elsewhere?

Reply:
I just wanted to keep this consistent. It's both in the `conda_environment.yml` and in `setup.py`, yes. If I don't add it, it feels like we're requiring different dependencies at different places which makes little sense?